source/_integrations/terncy.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: Terncy
+description: Instructions on how to setup Terncy devices within Home Assistant.
+ha_category:
+  - Light
+ha_iot_class: Local Push
+ha_release: '0.60'
+ha_config_flow: true
+ha_codeowners:
+  - '@rxwen'
+ha_domain: terncy
+---
+
+The Terncy integration allows you to control your Terncy devices connected to the Terncy Home Center (a Zigbee gateway) with Home Assistant.
+
+The Terncy integration is automatically discovered. If not, add it via the add integration menu.
```

Comment:
This line should be part of the configuration instructions. Furthermore, even automatically discovered needs to be added via the integrations menu.